server/src/config.js

```diff
@@ -5,5 +5,6 @@ export default {
   nodeEnv: getNodeEnv(),
   session: { secret: process.env.SESSION_SECRET },
   databaseUrl: getDatabaseUrl(getNodeEnv()),
-  web: { host: process.env.HOST || "0.0.0.0", port: process.env.PORT || 3000 }
+  web: { host: process.env.HOST || "0.0.0.0", port: process.env.PORT || 3000 },
+  maxAge: 1000 * 60 * 60 * 48
```

Comment:
I would set a `maxAgeHours` or something similar so this is an understandable data point.